package.json

```
@@ -57,7 +57,7 @@
     "remark-mdx": "^2.3.0",
     "remark-parse": "^10.0.1",
     "remark-stringify": "^10.0.3",
-    "transform-markdown-links": "^2.1.0",
+    "transform-images-links": "^2.1.0",
```

I think this was an accident

```suggestion
    "transform-markdown-links": "^2.1.0",
```